packages/big-design/src/components/Collapse/styled.tsx

```diff
@@ -0,0 +1,37 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { withTransition } from '../../mixins/transitions';
+import { StyleableButton } from '../Button/private';
+
+export const StyledButton = styled(StyleableButton).attrs<{}, { isOpen?: boolean }>({})`
+  padding-left: ${({ theme }) => theme.spacing.none};
+  padding-right: ${({ theme }) => theme.spacing.none};
+  width: auto;
+
+  &:active,
+  &:hover:not(:active) {
+    background: none;
+    color: ${({ theme }) => theme.colors.primary70};
+  }
+
+  &:focus {
+    box-shadow: none;
+  }
```

Comment:
This is not good at all; breaks accessibility completely. There needs to be some focus style.

Reply:
Discussed and fixed